sift/packages/avfs.sls

```diff
@@ -0,0 +1,2 @@
+avfs:
```

Comment:
please change this to

```
sift-package-avfs:
  pkg.installed:
    - name: avfs
```

Please repeat for all other packages. The first line is the SLS ID --- to prevent conflicts, these have to be globally unique, so if we run remnux and sift at the same time, we can't have to identical SLS IDs therefore a prefix should be used.